build.gradle

```diff
@@ -81,10 +111,69 @@ clean {
     return path
 }*/
 
-publishing {
-    publications {
-        mavenJava(MavenPublication) {
-            artifact jar
+// Configure publishing to the Sonatype Maven Central repo
+uploadArchives {
```

Comment:
'uploadArchives' comes from the 'maven' plugin, and is ancient and deprecated.  You should be using the 'publishing' configuration which is provided by the 'maven-publish' plugin.

Reply:
Hmm, strange that Maven Central says to use uploadArchives in their tutorial:  https://central.sonatype.org/pages/gradle.html

I see the gradle documentation on the new approach so I will update it to use that:  https://docs.gradle.org/current/userguide/publishing_setup.html#publishing_overview

Reply:
Ok, how does it look now?